Cargo.toml

```diff
@@ -20,6 +20,7 @@ members = [
   "crates/primitives",
   "modules/rust-wasm-test",
   "modules/benchmarks",
+  "modules/benchmarks_db",
```

Comment:
I don't see why this can't just go in the `benchmarks` module. At the very least you should create a README which explains what the difference between `benchmarks` and `benchmarks_db` is because it's not at all obvious based on the name.

Reply:
Ok, I have refactor the bench to be in the same module

Reply:
benchmarks please :)